tools/clang/test/HLSLFileCheck/passes/hl/sroa_hlsl/NodeInput_type.ll

```diff
@@ -8,6 +8,22 @@
 ; CHECK: alloca %"struct.DispatchNodeInputRecord<loadStressRecord>"
 ; CHECK: load %"struct.DispatchNodeInputRecord<loadStressRecord>", %"struct.DispatchNodeInputRecord<loadStressRecord>"*
```

Comment:
It's not strictly related, but given there is another nodeio type with an alloca `%agg.tmp` and load here, I think we ought to check for them as well.